docs/api/nbs-api-specification.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: NBS Interface Specification
+parent: API
+nav_order: 5
+---
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+# NBS API Specification
+
+To support interoperabilty with NBS, the Sara Alert API provides an endpoint that can be used to export monitorees in the Public Health Document Container (PHDC) format.
+
+<a name="supported-scopes"/>
+
+## Supported Scopes
+This endpoint is only supported for applications following the [SMART on FHIR Backend Services Workflow](api-getting-started#backend-services), and applications should follow the process detailed in that section to authenticate. This endpoint is only supported for the following scopes:
+
+* `system/Patient.read`,
+* `system/Patient.*`, (for both read and write access to this resource)
```

Comment:
Just need to add here that the `system/QuestionnaireResponse.read` scope is also necessary

Reply:
Oops! Fixed in https://github.com/SaraAlert/SaraAlert/pull/823/commits/68b0556cab6ee31008abad5dfb96c0c5acc6aeb1.